modules/standard/AutoMath.chpl

```diff
@@ -1929,7 +1959,7 @@ module AutoMath {
 
   /* Returns the maximum value of two arguments using the ``>`` operator
      for comparison.
-     If one of the arguments is :proc:`AutoMath.NAN`, the result is also NAN.
+     If one of the arguments is :proc:`AutoMath.nan`, the result is also NaN.
```

Comment:
If possible I would avoid using the "NaN" spelling, even in documentation, to avoid confusion.